mava/components/jax/building/parameter_client.py

```diff
@@ -50,19 +61,25 @@ def __init__(
         self,
         config: ExecutorParameterClientConfig = ExecutorParameterClientConfig(),
     ) -> None:
-        """Parameter client
+        """Component creates a parameter client for the executor.
 
         Args:
-            config : parameter client config
+            config: ExecutorParameterClientConfig.
         """
 
         self.config = config
 
     def on_building_executor_parameter_client(self, builder: SystemBuilder) -> None:
-        """_summary_
+        """Create and store the executor parameter client.
+
+        Gets network parameters from store and registers them for tracking.
```

Comment:
"registers them for tracking" is not so clear imo

Reply:
Changed to "for updating"

Reply:
Thanks